NMS/Common/src/main/java/com/fren_gor/ultimateAdvancementAPI/nms/wrappers/VanillaAdvancementDisablerWrapper.java

```diff
@@ -30,7 +31,7 @@ public class VanillaAdvancementDisablerWrapper {
      *
      * @throws Exception If disabling goes wrong.
      */
-    public static void disableVanillaAdvancements() throws Exception {
-        method.invoke(null);
+    public static void disableVanillaAdvancements(boolean disableVanillaAdvancementsRecipes) throws Exception {
```

Comment:
When fixing the breaking changes in NMS, you could add the method with no parameters *only* in this class and make it call this method (passing `true` as default value), since the NMS implementations are not exposed APIs and breaking changes are acceptable there.

Reply:
Good idea, done.